integrations/tensorflow/e2e/slim_vision_models/slim_vision_model_test.py

```diff
@@ -29,20 +29,19 @@
 # Testing vision models from
 # https://github.com/tensorflow/models/tree/master/research/slim
 # slim models were designed with tf v1 and then coverted to SavedModel
-# they are stored at tensorflow_hub. At least a subset can be viewed here:
-# https://tfhub.dev/s?dataset=imagenet&module-type=image-classification,image-classifier
+# they are stored at tensorflow_hub.
 flags.DEFINE_string(
-    'model_tag', 'mobilenet_v1_100_224', 'examples of model tag: '
-    'resnet_v1_50, resnet_v1_101, resnet_v2_50, resnet_v2_101, '
+    'model', 'mobilenet_v1_100_224', 'example model names: '
+    '[resnet_v1_50, resnet_v1_101, resnet_v2_50, resnet_v2_101, '
     'mobilenet_v1_100_224, mobilenet_v1_025_224, mobilenet_v2_100_224, '
-    'mobilenet_v2_035_224')
-
-HUB_URL = 'https://tfhub.dev/google/imagenet/'
+    'mobilenet_v2_035_224]\nAt least a subset can be viewed here:\n'
+    'https://tfhub.dev/s?dataset=imagenet&module-type=image-classification,image-classifier'
+)
+flags.DEFINE_string('hub_url', 'https://tfhub.dev/google/imagenet/',
+                    'Base URL for the models to test')
```

Comment:
But then pass it in as a flag as well :-D

The idea is that external dependencies should be really obvious when you're running something and required to be explicit. So you only take on some external server dependency if you're passing some explicit command line arguments. This is a dependency, so it should be in the build file. Maybe I'm just letting too much Google production best practices bleed into our little python script :grin:

Reply:
No, I think that's a fair point haha. Fixed.

Reply:
Last thing is defaulting the URL to the empty string and adding error checking.

Reply:
I'm going to leave the default so that it's still easy to use `bazel run` on the `_manual` targets.

Reply:
You can set it in the args on the `_manual` targets as well

Reply:
Good point! I forgot that this wasn't using the same list comprehension as `e2e/BUILD`. I defaulted it to None so that `posixpath.join` returns an error.